Index: Websites/AdminExtendedNauEdu/SiteRoot/Apps/Areas/TSC/Controllers/ManagerController.cs
===================================================================
--- Websites/AdminExtendedNauEdu/SiteRoot/Apps/Areas/TSC/Controllers/ManagerController.cs	(revision 29085)
+++ Websites/AdminExtendedNauEdu/SiteRoot/Apps/Areas/TSC/Controllers/ManagerController.cs	(working copy)
@@ -1,4 +1,7 @@
-﻿using System.Linq;
+﻿using System;
+using System.Data.Entity.Core.Objects;
+using System.Data.Entity.Infrastructure;
+using System.Linq;
 using System.Text.RegularExpressions;
 using System.Web.Mvc;
 using Application.Model.Dls;
@@ -10,12 +13,12 @@
 {
     public class ManagerController : BaseController
     {
-        private TscDbEntities db = new TscDbEntities();        
+        private TscDbEntities db = new TscDbEntities();
 
         // GET: TSC/Manager
         public ActionResult Index()
         {
-            var viewModel = new ManagerViewModel();
+            var viewModel = new ManagerViewModel();            
 
             // If the user is not a TSC, send them back to the requester view OR if they're not a manager
             var currentTsc = db.Tsc_Tsc.FirstOrDefault(u => u.Assignee_ID == User.EmplID.ToString());            
@@ -38,8 +41,7 @@
             else
             {
                 return RedirectToAction("Index", "Requester");
-            }
-
+            }            
             return View(viewModel);
         }
 
@@ -48,7 +50,6 @@
         [ValidateAntiForgeryToken]
         public HttpStatusCodeResult Save(int? id, string campusList)
         {
-            //Variable declaration 
             var viewModel = new ManagerViewModel();
 
             campusList = Regex.Replace(campusList, @"[^a-zA-Z0-9\s]", "");
@@ -100,6 +101,69 @@
             return new HttpStatusCodeResult(200);
         }
 
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public HttpStatusCodeResult SaveTSCs(string activeTSCs, string assigneeID)
+        {
+            activeTSCs = Regex.Replace(activeTSCs, @"[^a-zA-Z0-9\s]", "");
+
+            string[] previousActiveTSCs = db.Tsc_Tsc
+                .Where(x => x.Status == "A")
+                .Select(c => c.Tsc_ID.ToString())
+                .ToArray();
+
+            string[] ActiveTSCs = activeTSCs.Split();
+            ActiveTSCs = ActiveTSCs.Take(ActiveTSCs.Length - 1).ToArray();
+
+            foreach (var oldTSC in previousActiveTSCs)
+            {
+                if (!ActiveTSCs.Contains(oldTSC))
+                {
+                    Tsc_Tsc tscToChange = new Tsc_Tsc();
+                    tscToChange = db.Tsc_Tsc.Find(Convert.ToInt32(oldTSC));
+                    tscToChange.Status = "I";                                        
+                    db.SaveChanges();
+                }
+            }
+
+            foreach (var activeTSC in ActiveTSCs)
+            {
+                if (!previousActiveTSCs.Contains(activeTSC))
+                {
+                    Tsc_Tsc tscToChange = new Tsc_Tsc();
+                    tscToChange = db.Tsc_Tsc.Find(Convert.ToInt32(activeTSC));
+                    tscToChange.Status = "A";                                        
+                    db.SaveChanges();
+                }
+            }
+
+            if (assigneeID != null)
+            {
+                Tsc_Tsc tscToAdd = new Tsc_Tsc();
+                tscToAdd.Assignee_ID = assigneeID;
+                tscToAdd.Manager = "N";
+                tscToAdd.Status = "A";
+                db.Tsc_Tsc.Add(tscToAdd);
+                db.SaveChanges();
+            }
+
+            return new HttpStatusCodeResult(200);
+        }
+
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public string CheckTSC(string assigneeID)
+        {
+            string name = "";
+
+            if (assigneeID != null)
+            {
+                name = db.vwTsc_PS_Employees.Where(tsc => tsc.EMPLID == assigneeID).ToString();
+            }
+
+            return name;
+        }
+
         protected override void Dispose(bool disposing)
         {
             if (disposing)
Index: Websites/AdminExtendedNauEdu/SiteRoot/Apps/Areas/TSC/CSS/TscStyles.css
===================================================================
--- Websites/AdminExtendedNauEdu/SiteRoot/Apps/Areas/TSC/CSS/TscStyles.css	(revision 29085)
+++ Websites/AdminExtendedNauEdu/SiteRoot/Apps/Areas/TSC/CSS/TscStyles.css	(working copy)
@@ -147,4 +147,43 @@
     -moz-user-select: none;
     -ms-user-select: none;
     user-select: none;
+}
+
+#enterInformation {
+    background-color: #d9edf7;
+    border-color: #bce8f1;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    margin-left: 25%;
+    margin-right: 25%;
+}
+
+.modalHeader {
+    padding-top: 15px;
+    padding-left: 15px;
+    padding-right: 15px;
+}
+
+.modal-dialog {
+    width: 700px;
+}
+
+#checkAssignee {
+    border: double;
+    padding-top: 10px;
+    padding-bottom: 10px;
+}
+
+#modalBody {
+    margin-left: 20px;
+    margin-right: 20px;
+}
+
+#managerLabel {
+    text-align: left;
+}
+
+#checkTSC {
+    padding-top: 20px;
+    text-align: center;
 }
\ No newline at end of file
Index: Websites/AdminExtendedNauEdu/SiteRoot/Apps/Areas/TSC/Scripts/TscManager.js
===================================================================
--- Websites/AdminExtendedNauEdu/SiteRoot/Apps/Areas/TSC/Scripts/TscManager.js	(revision 29085)
+++ Websites/AdminExtendedNauEdu/SiteRoot/Apps/Areas/TSC/Scripts/TscManager.js	(working copy)
@@ -1,44 +1,115 @@
-﻿$(function () {
-    $('div').click(function () {
-        if ($(this).hasClass("campusTd")) {
-            $(this).toggleClass("blue-cell");
-        }
+﻿$(document).ready(function() {
+    $(function() {
+        $('div').click(function() {
+            if ($(this).hasClass("campusTd")) {
+                $(this).toggleClass("blue-cell");
+            }
+        });
     });
-});
 
-function saveData() {
-    var id = $(".in").attr("id");
-    var antiForgeryToken = $("#antiForgeryToken").text();
-    var campusList = "";
-    $('.in').find('.blue-cell').each(function (index) {
-        campusList += $(this).attr('id') + " ";
+    $(function() {
+        $('#addNewTSCButton').click(function () {
+            $('#newTSCDiv').show();
+            $('#activationTables').hide();
+        });
     });
 
-    var campuses = "";
-    $('.in').find('.blue-cell').each(function (index) {
-        campuses += $(this).attr('title') + "*";
+    $(function () {
+        $('#showTables').click(function () {
+            $('#activationTables').show();
+            $('#newTSCDiv').hide();
+        });
     });
 
-    $.ajax({
-        type: "post",
-        url: "/Apps/tsc/manager/save/",
-        data: {
-            __RequestVerificationToken: antiForgeryToken,
-            id: id,
-            campusList: JSON.stringify(campusList)
-        },
-        success: function () {
-            $("#" + id).modal('toggle');
+    $(function () {
+        $('#editTSCsButton').click(function () {
+            $('#activationTables').show();
+            $('#newTSCDiv').hide();
+        });
+    });
 
-            var campusHtml = "";
-            $.each(campuses.split("*"), function (index, value) {
-                campusHtml += (value + "<br>");
-            });
+    $('#chkTSC').click(function() {
+        var assigneeID = $('#assigneeID').Text;
+        var antiForgeryToken = $("#antiForgeryToken").text();
 
-            $("#campuses_" + id).html(campusHtml);
-        },
-        error: function () {
-            document.getElementById('errorMessage').style.display = "table-footer-group";
-        }
+        $.ajax({
+            type: "post",
+            url: "/Apps/tsc/manager/CheckTSC/",
+            data: {
+                __RequestVerificationToken: antiForgeryToken,
+                assigneeID: assigneeID
+            },
+            success: function (name) {
+                $("#Name").Text = name;
+            },
+            error: function () {
+                document.getElementById('errorMessageTSCs').style.display = "table-footer-group";
+            }
+        });
     });
-}
\ No newline at end of file
+
+    function saveData() {
+        var id = $(".in").attr("id");
+        var antiForgeryToken = $("#antiForgeryToken").text();
+        var campusList = "";
+        $('.in').find('.blue-cell').each(function(index) {
+            campusList += $(this).attr('id') + " ";
+        });
+
+        var campuses = "";
+        $('.in').find('.blue-cell').each(function(index) {
+            campuses += $(this).attr('title') + "*";
+        });
+
+        $.ajax({
+            type: "post",
+            url: "/Apps/tsc/manager/save/",
+            data: {
+                __RequestVerificationToken: antiForgeryToken,
+                id: id,
+                campusList: JSON.stringify(campusList)
+            },
+            success: function() {
+                $("#" + id).modal('toggle');
+
+                var campusHtml = "";
+                $.each(campuses.split("*"), function(index, value) {
+                    campusHtml += (value + "<br>");
+                });
+
+                $("#campuses_" + id).html(campusHtml);
+            },
+            error: function() {
+                document.getElementById('errorMessage').style.display = "table-footer-group";
+            }
+        });
+    }
+
+    $('.saveChange').click(function() {
+        var antiForgeryToken = $("#antiForgeryToken").text();
+        var activeTSCs = "";
+        var assigneeID = $('#assigneeID').Text;
+
+        $('#editTSCs').find('.statusOfA').each(function (index) {
+            if ($(this).hasClass('active')) {
+                activeTSCs += $(this).attr('title') + " ";
+            }
+        });
+
+        $.ajax({
+            type: "post",
+            url: "/Apps/tsc/manager/SaveTSCs/",
+            data: {
+                __RequestVerificationToken: antiForgeryToken,
+                activeTSCs: JSON.stringify(activeTSCs),
+                assigneeID: assigneeID
+    },
+            success: function() {
+                $("#editTSCs").modal('toggle');
+            },
+            error: function() {
+                document.getElementById('errorMessageTSCs').style.display = "table-footer-group";
+            }
+        });
+    });
+});
\ No newline at end of file
Index: Websites/AdminExtendedNauEdu/SiteRoot/Apps/Areas/TSC/Views/Manager/Index.cshtml
===================================================================
--- Websites/AdminExtendedNauEdu/SiteRoot/Apps/Areas/TSC/Views/Manager/Index.cshtml	(revision 29085)
+++ Websites/AdminExtendedNauEdu/SiteRoot/Apps/Areas/TSC/Views/Manager/Index.cshtml	(working copy)
@@ -9,6 +9,7 @@
 <link type="text/css" rel="stylesheet" href="~/Areas/TSC/CSS/TscStyles.css"/>
 <div class="container">
     <h2>TSC Manager</h2>
+    <button type="button" id="editTSCsButton" class="btn btn-primary pull-right" data-toggle="modal" data-target="#editTSCs"><span class="glyphicon glyphicon-pencil"></span> Edit TSC's</button>
     <ul class="nav nav-tabs" role="tablist">
         <li role="presentation"><a href="@Url.Action("Index", "Requester")" role="tab">Requester</a></li>
         <li role="presentation"><a href="@Url.Action("Index", "Tsc")" role="tab">Manage Tickets</a></li>
@@ -128,7 +129,7 @@
                             </div>
                             <div class="modal-footer">
                                 <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
-                                <button type="button" class="btn btn-primary" value="Save" onclick=" saveData() ">Save Changes</button>
+                                <button type="button" class="btn btn-primary" value="Save" onclick="saveData()">Save Changes</button>
                             </div>
                         </div>
                     </div>
@@ -136,6 +137,124 @@
             </div>
         }
     }
+    @*This Modal allows the User to edit the TSC's, which either activats or inactivates them*@
+    <div class="modal fade" id="editTSCs">
+        <form class="form-horizontal" role="form" method="post" action="">
+            @Html.AntiForgeryToken()
+            <div class="modal-dialog">
+                <div class="modal-content">
+                    <div class="modalHeader">
+                        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                        <h4 class="modal-title">Edit TSC's</h4>
+                    </div>
+                        <ul class="nav nav-tabs" role="tablist">
+                            <li role="presentation" id="showTables"><a href="#" role="tab">Edit TSC's</a></li>
+                            <li role="presentation" id="addNewTSCButton"><a href="#" role="tab">Add New TSC</a></li>
+                        </ul>
+                    <div class="modal-body" id="modalBody">
+                        <div id="activationTables">
+                        @if (Model.TSCs.Any(tsc => tsc.Status == "A"))
+                        {
+                            <table class="table condensed" id="TSCTableActive">
+                                <tr><th>Active Technical Support Coordinator(s)</th><th style="text-align: center;">Status</th><th>Manager</th></tr>
+                                @foreach (var TSC in Model.TSCs)
+                                {
+                                    if (TSC.Status == "A")
+                                    {
+                                        <tr>
+                                            <td>@Html.DisplayFor(modelItem => modelItem.AllAdminUsersList.First(u => u.Value == TSC.Assignee_ID).Text)</td>
+                                            <td style="text-align: center;">
+                                                <div class="btn-group" data-toggle="buttons">
+                                                    <label class="btn btn-primary statusOfI" title="@TSC.Tsc_ID">
+                                                        <input type="radio" name="options" autocomplete="off"> Inactive
+                                                    </label>
+                                                    <label class="btn btn-primary active statusOfA" title="@TSC.Tsc_ID">
+                                                        <input type="radio" name="options" autocomplete="off"> Active
+                                                    </label>
+                                                </div>
+                                            </td>
+                                            <td style="text-align: center;">
+                                                @if (TSC.Manager == "Y")
+                                                {
+                                                    <input type="checkbox" name="managerStatus" checked>
+                                                }
+                                                else
+                                                {
+                                                    <input type="checkbox" name="managerStatus">
+                                                }
+                                            </td>
+                                        </tr>
+                                    }
+                                }
+                            </table>
+                        }
+                        @if (Model.TSCs.Any(tsc => tsc.Status == "I"))
+                        {
+                            <table class="table condensed" id="TSCTableInactive">
+                                <tr><th>Inactive Technical Support Coordinator(s)</th><th style="text-align: center;">Status</th><th>Manager</th></tr>
+                                @foreach (var TSC in Model.TSCs)
+                                {
+                                    if (TSC.Status == "I")
+                                    {
+                                        <tr>
+                                            <td>@Html.DisplayFor(modelItem => modelItem.AllAdminUsersList.First(u => u.Value == TSC.Assignee_ID).Text)</td>
+                                            <td style="text-align: center;">
+                                                <div class="btn-group" data-toggle="buttons">
+                                                    <label class="btn btn-primary active statusOfI" title="@TSC.Tsc_ID">
+                                                        <input type="radio" name="options" autocomplete="off"> Inactive
+                                                    </label>
+                                                    <label class="btn btn-primary statusOfA" title="@TSC.Tsc_ID">
+                                                        <input type="radio" name="options" autocomplete="off"> Active
+                                                    </label>
+                                                </div>
+                                            </td>
+                                            <td style="text-align: center;">
+                                                @if (TSC.Manager == "Y")
+                                                {
+                                                    <input type="checkbox" name="managerStatus" checked>
+                                                }
+                                                else
+                                                {
+                                                    <input type="checkbox" name="managerStatus">
+                                                }
+                                            </td>
+                                        </tr>
+                                    }
+                                }
+                            </table>
+                        }
+                        </div>
+                        <div id="newTSCDiv">
+                            <div class="row" id="enterInformation">
+                                <label class="col-md-5 control-label">Employee ID: </label>
+                                <div class="col-md-5">
+                                    <input type="text" class="form-control" id="assigneeID" placeholder="1234567">
+                                </div>
+                                @*<label class="col-md-4 control-label" id="managerLabel">Manager? <input type="checkbox" value="manager"></label>*@
+                            </div>
+                            <div class="row" id="checkTSC">
+                                <div class="col-md-4 col-md-offset-4">
+                                    <button type="button" class="btn btn-primary" id="chkTSC">Check TSC</button>
+                                </div>
+                            </div>
+                                <br />
+                                <div class="row" id="checkAssignee">
+                                    <label class="col-sm-2 col-sm-offset-3 control-label">Name: </label>
+                                    <div class="col-sm-4">
+                                        <input type="text" class="form-control" id="Name" disabled>
+                                    </div>
+                                </div>
+                            </div>
+                        <p id="errorMessageTSCs" style="display: none; color: #FF0000; position: absolute; right: 5px; width: 275px;">There was an error saving the TSC(s).</p><br/>
+                    </div>
+                    <div class="modal-footer">
+                        <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+                        <button type="button" class="btn btn-primary saveChange">Save changes</button>
+                    </div>
+                </div>
+            </div>
+        </form>
+    </div>
 </div>
 <div id="antiForgeryToken" class="hidden">@Html.AntiForgeryTokenForAjaxPost()</div>
 <script src="~/Areas/TSC/Scripts/TscManager.js"></script>
