Index: Websites/AdminExtendedNauEdu/SiteRoot/Apps/Areas/TSC/Scripts/TicketFunctions.js
===================================================================
--- Websites/AdminExtendedNauEdu/SiteRoot/Apps/Areas/TSC/Scripts/TicketFunctions.js	(revision 29916)
+++ Websites/AdminExtendedNauEdu/SiteRoot/Apps/Areas/TSC/Scripts/TicketFunctions.js	(working copy)
@@ -1197,6 +1197,7 @@
         $("#ddlSortOrder").html('<span class="glyphicon glyphicon-chevron-down"></span> Sort By');
         $("#ddlTicketStatus").html('<span class="glyphicon glyphicon-chevron-down"></span> Status');
         $("#ddlCampus").html('<span class="glyphicon glyphicon-chevron-down"></span> Campuses');
+        $(".worklog td").css("width", "auto");
     }
     if ($(window).width() >= 768 && $(window).width() < 992) {
         $("#ddlSortOrder").html('<span class="glyphicon glyphicon-chevron-down"></span> Sort By');
Index: Websites/AdminExtendedNauEdu/SiteRoot/Apps/Areas/TSC/Views/Requester/_RequesterTicketList.cshtml
===================================================================
--- Websites/AdminExtendedNauEdu/SiteRoot/Apps/Areas/TSC/Views/Requester/_RequesterTicketList.cshtml	(revision 29916)
+++ Websites/AdminExtendedNauEdu/SiteRoot/Apps/Areas/TSC/Views/Requester/_RequesterTicketList.cshtml	(working copy)
@@ -119,9 +119,9 @@
                         @foreach (var workLogEntry in ticket.Tsc_Work_Log)
                         {
                             <tr>
-                                <td style="width: auto">@workLogEntry.Date_Worked</td>
-                                <td style="width: auto">@workLogEntry.Work</td>
-                                <td style="width: auto">
+                                <td style="width: 10%">@workLogEntry.Date_Worked.ToString("MM/dd/yyyy")</td>
+                                <td style="width: 70%">@workLogEntry.Work</td>
+                                <td style="width: 20%">
                                     @if (Model.Employees.Any(u => u.EMPLID == workLogEntry.Assignee_ID))
                                     {
                                         @Model.Employees.First(u => u.EMPLID == workLogEntry.Assignee_ID).NAME
Index: Websites/AdminExtendedNauEdu/SiteRoot/Apps/Areas/TSC/Views/TSC/_TSCTicketList.cshtml
===================================================================
--- Websites/AdminExtendedNauEdu/SiteRoot/Apps/Areas/TSC/Views/TSC/_TSCTicketList.cshtml	(revision 29916)
+++ Websites/AdminExtendedNauEdu/SiteRoot/Apps/Areas/TSC/Views/TSC/_TSCTicketList.cshtml	(working copy)
@@ -189,9 +189,9 @@
                 @foreach (var workLogEntry in logs)
                 {
                     <tr>
-                        <td style="width: auto" class="log-date">@workLogEntry.Date_Worked</td>
-                        <td style="width: auto">@workLogEntry.Work</td>
-                        <td style="width: auto">
+                        <td style="width: 10%" class="log-date">@workLogEntry.Date_Worked.ToString("MM/dd/yyyy")</td>
+                        <td style="width: 70%">@workLogEntry.Work</td>
+                        <td style="width: 20%">
                             @if (Model.Employees.Any(u => u.EMPLID == workLogEntry.Assignee_ID))
                             {
                                 @Model.Employees.First(u => u.EMPLID == workLogEntry.Assignee_ID).NAME
